api/route/route.go

```diff
@@ -480,6 +482,58 @@ func findRoute(c *gin.Context) {
 	}
 }
 
+func getRouteAndApisixUrl(c *gin.Context) {
```

Comment:
could we use camelCase here? e.g `getRouteWithAPISIXURL` .

Reply:
Refer to the naming of Apisix in other documents, updated to getRouteWithApisixUrl :)